lib/restoreYarn.js

```diff
@@ -0,0 +1,15 @@
+import { exec } from "@actions/exec";
+
+/**
+ * @param {import("@actions/exec").ExecOptions} [options]
+ * @returns {Promise<void>}
+ */
+export default async function restoreYarn(options = {}) {
+  const cwd = options.cwd || process.cwd();
+
+  await exec("rm -rf yarn.lock", undefined, { ...options, cwd });
+
+  await exec("yarn import --silent", undefined, { ...options, cwd });
+
+  await exec("rm -rf package-lock.json", undefined, { ...options, cwd });
+}
```

Comment:
[suggestion]

- Please use `fs` instead of `rm -rf`, just in case.
- Please use an argument array for `exec()`.

For example:

```js
import * as fs from "fs";

/**
 * TODO: `fs.rm()` is available since Node.js v14.14.0.
 *
 * @param {string} filePath
 */
async function forceDelete(filePath) {
  if (fs.existsSync(filePath)) {
    await fs.promises.unlink(filePath)
  }
}

/**
 * @returns {Promise<void>}
 */
export default async function restoreYarn() {
  await forceDelete("yarn.lock");

  await exec("yarn", ["import", "--silent"]);

  await forceDelete("package-lock.json");
}
```